doc/manipulating_images/masker_objects.rst

```diff
@@ -165,7 +165,9 @@ The `mask_strategy` argument controls how the mask is computed:
 
 * `background`: detects a continuous background
 * `epi`: suitable for EPI images
-* `template`: uses an MNI grey-matter template
+* `wb-template`: uses an MNI whole-brain template
```

Comment:
to me `wb-template` is not meaningful to people. I have nothing better to propose than `whole-brain-template`